pubtools/_quay/operator_pusher.py

```diff
@@ -326,6 +324,39 @@ def iib_remove_operators(cls, operators=None, archs=None, index_image=None, targ
             env_vars,
         )
 
+    def get_existing_index_images(self, quay_client):
+        """
+        Return existing index images for push items.
+
+        Args:
+            quay_client (QuayClient): quay_client_instance
+
+        Returns [(digest, tag)]:
+            List of tuples containing digest and tag of existing index image
+        """
+        image_schema = "{host}/{namespace}/{repo}"
+        iib_repo = get_internal_container_repo_name(
+            self.target_settings["quay_operator_repository"]
+        )
+        index_image_repo = image_schema.format(
+            host=self.quay_host, namespace=self.target_settings["quay_namespace"], repo=iib_repo
+        )
+        current_index_images = []
+
+        manifest_list = {}
+        for version in sorted(self.version_items_mapping.keys()):
+            image_ref = "{0}:{1}".format(index_image_repo, version)
+            try:
+                manifest_list = quay_client.get_manifest(image_ref, manifest_list=True)
+            except requests.exceptions.HTTPError as e:
+                if e.response.status_code == 404:
+                    continue
+                else:
+                    raise
+            for manifest in manifest_list["manifests"]:
+                current_index_images.append((manifest["digest"], version))
```

Comment:
It seems like the tag is not used anywhere. Is there a need to return it?

Reply:
It's not necessary to return it for purpose of cleaning up outdated signatures